schema/pgsql/schema.sql

```diff
@@ -146,61 +152,67 @@ CREATE TABLE rotation (
     changed_at bigint NOT NULL,
     deleted boolenum NOT NULL DEFAULT 'n',
 
+    CONSTRAINT pk_rotation PRIMARY KEY (id),
+
     -- Each schedule can only have one rotation with a given priority starting at a given date.
     -- Columns schedule_id, priority, first_handoff must be NULLed for deletion via "deleted = 'y'".
-    UNIQUE (schedule_id, priority, first_handoff),
-    CHECK (deleted = 'y' OR priority IS NOT NULL AND first_handoff IS NOT NULL),
+    CONSTRAINT uk_rotation_schedule_schedule_priority_first_handoff UNIQUE (schedule_id, priority, first_handoff),
```

Comment:
I don't see where the second "schedule" in "uk_rotation_**schedule_schedule**_priority_first_handoff" is coming from.